datapool.go

```diff
@@ -306,7 +319,7 @@ func (dp *datapool) pushKeys(point_time time.Time) (int, int) {
 		pointData := tsdPoint.data
 		currentFileInfo := dp.last_time_file[tsdPoint.filename]
 
-		if dp.lg.stale_removal && pointData.Stale(point_time) {
+		if dp.lg.stale_removal && pointData.Stale(point_time) && !tsdPoint.never_stale {
```

Comment:
Wouldn't `stale_removal = false` as config accomplish the same thing ?

Reply:
I suspect this one was to allow us to be a bit more fine-grained with stale removal (ie; at a per metric level, rather than all metrics?)

Reply:
Oh right, missed that. Great then!